src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(prey) {
+    if (prey.constructor.name === 'Herbivore' && !prey.hidden) {
+      prey.health -= 50;
+    }
+
+    if (prey.health <= 0) {
```

Comment:
Normally I would not warn about this but I was rejected with this. 😄
Because health is fixed 100 in the beginning and we remove 50 when bit we can just
`prey.health === 0`